test/Exporter/Test/Writer/FormattedBoolWriterTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
```

Comment:
@rande Are you talking of the header with your contact and copyright which is present in the Writer as comment?